x/claim/client/cli/tx_claim_eth.go

```diff
@@ -0,0 +1,44 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/arkeonetwork/arkeo/x/claim/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
```

Comment:
same question as above , not sure what this is for?

Reply:
same...

Reply:
[b617e11](https://github.com/arkeonetwork/arkeo/pull/29/commits/b617e11aeb8603c11f0eaeac1a0b286397ef05cd)